### ISSUE TITLE
arm64 build

### ISSUE THREAD

**Issue by LilDrunkenSmurf** [morphy2k/rss-forwarder#175]
Currently I see that the rss-forwarder is x86_64 only. Is there a possibility of adding an image build for arm64?

**Comment by morphy2k** [comment]
For which platform exactly? Linux?

**Comment by LilDrunkenSmurf** [comment]
Yes. I'm trying to run this on a raspberry pi.

**Comment by morphy2k** [comment]
I could cross-compile, but unfortunately I have no way of testing it on the platform to make sure it really works.

I will take a closer look at this when I get the chance. Alternatively, you can install Rust on your Raspberry Pi and compile it yourself.

```BASH
# Install the Rust toolchain
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh

# Compile and install rss-forwarder
cargo install rss-forwarder
```

Let me know if it works.

**Comment by LilDrunkenSmurf** [comment]
Tried building on 0.6.1, and got this:
```rss-forwarder: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory```

Tried building on 0.7.0-beta.2 and got this:
```rss-forwarder: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by rss-forwarder)```

Using this dockerfile:
https://github.com/LilDrunkenSmurf/containers/blob/main/apps/rss-forwarder/Dockerfile

**Comment by morphy2k** [comment]
I just noticed that I forgot to update the Dockerfile that you probably used as a reference.

[Line 1](https://github.com/LilDrunkenSmurf/containers/blob/d844c9cc7a85f860199e527d4e619b48d33786a0/apps/rss-forwarder/Dockerfile#L1)
```diff
- FROM rust:1.75.0 as builder
+ FROM rust:1.75.0-bookworm as builder
```
and [line 26](https://github.com/LilDrunkenSmurf/containers/blob/d844c9cc7a85f860199e527d4e619b48d33786a0/apps/rss-forwarder/Dockerfile#L26)
```diff
- FROM gcr.io/distroless/cc-debian11
+ FROM gcr.io/distroless/cc-debian12
```
needs to be updated

**Comment by LilDrunkenSmurf** [comment]
That worked! It's happily running on my RPis. Thanks!

**Comment by morphy2k** [comment]
There will probably be an official ARM64 image and a binary from the next release. See #181

**Comment by morphy2k** [comment]
I have just released **v0.7.0-beta.3**, which comes with a linux/arm64 image. In addition, a binary is included with the release.

**Comment by LilDrunkenSmurf** [comment]
I can confirm the image is working wonderfully on arm64. Thanks so much!
